recipes/xframe/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "xframe" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "8a33f4cd1ddb62d67ec59cb96c71f2c9396ddfc83e3580a514bbc09a5dd387bb" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/QuantStack/xframe/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1000
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - xtensor >=0.19.1,<0.20
+  run:
+    - xtensor >=0.19.1,<0.20
```

Comment:
I would just do `{{ pin_compatible('xtensor', max_pin='x.x') }}`, so that it requires the version of xtensor used at build time, up to a greater patch version.

Reply:
This is also fine like this. We should get this in a-is and make the change for 0.0.2.